### PR TITLE
fix(react-components): Fix update issue when open SettingsButton and fix slow hover on 360 images

### PR DESCRIPTION
**Pull request by nilscognite**
#### Type of change
Bug 

## Description :pencil:

* New version 0.64.2
* Rename onHover to onHoverByDebounce, since it was like that.
* Make a new onHover  without debounce. 
* Use the new on NavigationTool to hover on the 360 markers.
* Fix updating issue when open settingsbutton. 
* Fix closing when click on open settingsbutton.

## How has this been tested? :mag:

1. Open settingsbutton with a point cloud location. Check that it has all settings
2. Show some 360 markers and hover around. 

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [ ] I have performed a self-review of my own code.
- [ ] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/architecture/base/commands/BaseTool.ts [hunk 1]
```diff
@@ -85,7 +85,14 @@ export abstract class BaseTool extends RenderTargetCommand {
     // Override this to clear any temporary objects in the tool, like the dragger
   }
 
-  public onHover(_event: PointerEvent): void {}
+  public onHover(_event: MouseEvent): void {
+    // Debounce version, fast. Use this for hover effects when not
```
Comment (haakonflatval-cognite) [inline react-components/src/architecture/base/commands/BaseTool.ts:89]: Is this also supposed to say `Debounce version`?

File: react-components/src/architecture/base/concreteCommands/NavigationTool.ts [hunk 2]
```diff
@@ -33,10 +33,14 @@ export class NavigationTool extends BaseTool {
...
   }
 
-  public override onHover(event: PointerEvent): void {
+  public override onHoverByDebounce(event: PointerEvent): void {
     this.renderTarget.viewer.onHover360Images(event);
   }
 
+  public override onHover(event: MouseEvent): void {
+    this.renderTarget.viewer.onHover360Images(event as PointerEvent);
+  }
```
Comment (haakonflatval-cognite) [inline react-components/src/architecture/base/concreteCommands/NavigationTool.ts:42]: Should we call `onHover360Images` in both methods?

Comment (haakonflatval-cognite) [reply react-components/src/architecture/base/concreteCommands/NavigationTool.ts:42]: Also - should we fix the `PointerEvent` <-> `MouseEvent` issue in the function signature?

**Review by haakonflatval-cognite — APPROVED** [review]
LGTM :+1:

File: react-components/src/architecture/base/commands/BaseTool.ts (hunk 1 above)
Comment (nilscognite) [reply react-components/src/architecture/base/commands/BaseTool.ts]: Changed

File: react-components/src/architecture/base/concreteCommands/NavigationTool.ts (hunk 2 above)
Comment (nilscognite) [reply react-components/src/architecture/base/concreteCommands/NavigationTool.ts]: It should be MouseEvent. It is the old one that should have MouseEvent

**Pull request merged**